langfuse/model.py

```diff
@@ -151,11 +158,13 @@ def get_langchain_prompt(self):
 class ChatPromptClient(BasePromptClient):
     def __init__(self, prompt: Prompt_Chat):
         super().__init__(prompt)
-        self.prompt = prompt.prompt
+        self.prompt = [
+            ChatMessageDict(role=p.role, content=p.content) for p in prompt.prompt
+        ]
 
-    def compile(self, **kwargs) -> List[ChatMessage]:
+    def compile(self, **kwargs) -> List[ChatMessageDict]:
```

Comment:
Is this a breaking change if someone imported `ChatMessage` originally? I think yes. The import path would have involved a path to the api module, correct?
If yes, this is hard to change, if no, i would suggest to keep the naming of the exported package.